assets/src/dashboard/theme.js

```diff
@@ -214,6 +220,10 @@ const theme = {
   pageGutter: {
     desktop: 20,
     min: 10,
+    large: {
+      desktop: 80,
+      tablet: 40,
+    },
```

Comment:
The gutters for the detail page (according to the figma designs) are particularly large, so I created a new set of pageGutters... does this make sense?  And is `large` a good signifier?  Or should I treat "detail page" as a separate thing, like should it be: `detailPageGutter`?

Reply:
I wouldn't break it up. But I think we could give the 20, 10 values a label like small so it's clear the intent.

Reply:
I like it, I'll do it!